dev_guide/service_accounts.adoc

```diff
@@ -213,6 +213,35 @@ Tokens:            	robot-token-8bhpp
 ----
 ====
 
+== Checking if Pod Creation is Possible
```

Comment:
Both of the commands accept Pods and any other type of resources that embeds Pod/PodTemplate. I agree with a clear description what each of the commands should be used for, before diving into examples.